samza-yarn/src/main/java/org/apache/samza/job/yarn/YarnContainerRunner.java

```diff
@@ -190,13 +196,24 @@ private void startContainer(Path packagePath,
       throw new SamzaContainerLaunchException("IO Exception when writing credentials to output buffer");
     }
 
+    Map<String, LocalResource> localResourceMap = new HashMap<>();
```

Comment:
I'm fine if this is not done right away. We agree that once we deprecate `yarn.package.path`, the code path will be unified as per  my previous comment.